### PR TITLE
fix: Oracle response of OracleRespondTx transactions is not displayed correctly

### PR DESCRIPTION
**Pull request by janmichek**
## Description
This pr has been forgotten to merge in previous MVP private repo, so just moving it here

## Demo
Before 
image

After
image

## Checklist:
- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] [truncate Oracle respond data in diffrent way

### CODE REVIEW

File: src/components/TransactionCellOracleRespondTx.vue
Comment (lukeromanowicz) [inline src/components/TransactionCellOracleRespondTx.vue:2]: Shouldn't we keep the oracle url?

Comment (janmichek) [reply src/components/TransactionCellOracleRespondTx.vue]: Yes we should, fixed

**Review by lukeromanowicz — APPROVED** [review]
Not sure if we should show the empty chip. Maybe we should just hide it if the response is empty.

Nonetheless, it's a good improvement - thanks!

**Comment by janmichek** [comment]
@michele-franchi Waiting for review from you

**Review by michele-franchi — CHANGES_REQUESTED** [review]
I had a look at the original issue and I think it's still not full solved.

As you can see from the screenshot, empty values are printed:
image

What do you think about hiding them?

In general what we discussed here back then was to find a way to get rid of the responses that are for example:
- []...[]
- ye...es
- 8...8

And If I remember correctly my suggestion was the following:
- Hide the value if it's empty
- Print the value when it's under 8 chars
- Use the hash value component if it's more than 8 chars


I also noticed, not really related to your fix, that some responses are not printed at all. As you can see in the screenshot the first rows should contain "good":
image

Feel free to open a separate issue if you think this is not the right place.

**Comment by janmichek** [comment]
I successfully resolved the ellipsing of short responses.

The other problem with `no response` is a bit more complicated.
The problem is that some responses are encoded and some are note.

I am not really sure if it's an issue as this appears only on testnet? Maybe It was a product of some experimentation? Really not sure here

File: src/components/TransactionCellOracleRespondTx.vue
Comment (michele-franchi) [inline src/components/TransactionCellOracleRespondTx.vue:30]:
```suggestion
  const { response } = props.transactionData
  const decodedResponse = response instanceof Array ? formatDecodeByteArray(response) : response.toString()
```

This should also handle empty cases, what do you think?

**Review by michele-franchi — COMMENTED** [review]
I added a suggestion for the empty cases, I tested locally with testnet and it works but feel free to adjust the code if you don't like it 😄

File: src/components/TransactionCellOracleRespondTx.vue
Comment (janmichek) [reply src/components/TransactionCellOracleRespondTx.vue:30]: Ok, good. I wanted to avoid fixing data inconsistencies on FE, but ok, I accept your suggestion. It's a bit better solution
However, it didn't solve it ultimately. There are still some characters that cannot be printed, looking like it's 'empty'

image

**Review by michele-franchi — APPROVED** [review]
Looks good now, great job 👏

**Pull request merged**
